content/docs/versions/1-7/fundamentals/key-concepts.md

```diff
@@ -20,6 +20,13 @@ To deploy a box, a custom resource (CR) must be created. Each CR is a `yaml` fil
 The CR contains information such as Docker image and version, resources, pins, etc. 
 A set of boxes (in the form of CRs) is provided in infra-schema in a Git repository.
 
+## External Box
+To set the external box option enable the `externalBox` field of a custom resource to `true`.
+With this option you can run any th2 box runtime on your local machine like it is already a part of the Kubernetes cluster.
```

Comment:
It is better to start definition with "Something is". 

Suggestion is not final variant. It can be improved.

```suggestion
External Box is the optional work mode of the box. Using this box in this mode you can run any th2 box runtime on your local machine like it is already a part of the Kubernetes cluster.
```